# SPDX-FileCopyrightText: Huawei Inc.
# SPDX-License-Identifier: Apache-2.0

Fix build with modern compilers, such as clang 12 and gcc 11

Signed-off-by: Bernhard Rosenkraenzer <bernhard.rosenkraenzer.ext@huawei.com>

diff -up git/src/stats.h.omv~ git/src/stats.h
--- git/src/stats.h.omv~	2021-03-11 18:38:55.022070855 +0100
+++ git/src/stats.h	2021-03-11 18:38:59.498751697 +0100
@@ -33,7 +33,7 @@ enum stats_e { STATS_TCP_ACCEPT, STATS_R
                 STATS_COMPILE_OK, STATS_COMPILE_ERROR, STATS_COMPILE_TIMEOUT,
                 STATS_CLI_DISCONN, STATS_OTHER, STATS_ENUM_MAX };
 
-const char *stats_text[20];
+extern const char *stats_text[20];
 
 int  dcc_stats_init(void);
 void dcc_stats_init_kid(void);
